imaginary/garments.py

```diff
@@ -81,6 +83,42 @@ class GarmentSlot:
 
 
 
+class Wear(TargetAction):
+    expr = (pyparsing.Literal("wear") +
+            pyparsing.White() +
+            targetString("target"))
+
+    targetInterface = iimaginary.IClothing
+    actorInterface = iimaginary.IClothingWearer
+
+    def do(self, player, line, target):
+        from imaginary import garments
+        try:
+            player.putOn(target)
+        except garments.TooBulky as e:
+            raise eimaginary.ActionFailure(events.ThatDoesntWork(
+                actor=player.thing,
+                target=target.thing,
+                actorMessage=language.Sentence([
+                    language.Noun(e.wornGarment.thing).definiteNounPhrase(),
+                    u" you are already wearing is too bulky for you to do"
+                    u" that."]),
+                otherMessage=language.Sentence([
+                    player.thing,
+                    u" wrestles with basic personal problems."])))
+
+        evt = events.Success(
```

Comment:
I understand that this is "just" moving the action to a different module, but while you are at it - Is there a specific reason why the name `evt` is used instead of something a slightly more descriptive?